docs/to-do.rst

```diff
@@ -19,13 +19,20 @@ Need to update these docs to reflect:
 * Document this new breakpoint functionality:
   https://github.com/open-mpi/ompi/commit/f97d081cf9b540c5a79e00aecee17b25e8c123ad
 
-* Document v5.0.x's ABI relationship with v4.x.  It will likely be:
+* Add that we *do* support "single dash" ``mpirun`` options, but we
+  don't want people to do that.
+
+* Document v5.0.x's backwards-compatibility relationship with v4.x
+  (including ABI and ``mpirun`` command line options).  It will likely
+  be:
 
   * C bindings are ABI compatible
+    * Fortran ABI has changed.
```

Comment:
The mpif.h bindings should still be ABI compatible.  The use_mpi variants are changed, I think?

Reply:
I noted this in the to-do list.

Reply:
I noted this in the to-do list.